app/Http/Controllers/API/TraderController.php

```diff
@@ -114,7 +114,12 @@ public function showVouchers(Trader $trader)
                 ]);
             case 'application/json':
             default:
-                return response()->json($vouchers, 200);
+                return response()->json($vouchers->map(function($voucher){
```

Comment:
Or rather than return Voucher objects wholly just get the bits you want and make a new array. This was a quick auto-implementation before we know the structure of the json needed by the app.